.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I guess that we don't need to add this file

Reply:
ah right, it was not meant to be committed 🤦‍♂

Reply:
remove in 41d5fe2